packages/core/strapi/lib/middlewares/compression.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const compress = require('koa-compress');
+
+/**
+ * @type {import('./').MiddlewareFactory}
+ */
+module.exports = (options = {}) => compress(options);
```

Comment:
Why do we need to encapsulate the koa-compress export  (I would say to pass a default value to options but compress already handles it according to the typings)